sdk/communication/azure-communication-rooms/src/test/java/com/azure/communication/rooms/RoomsTestBase.java

```diff
@@ -168,32 +175,45 @@ protected void createUsers(HttpClient httpClient) {
         secondParticipantId = communicationClient.createUser();
         thirdParticipantId = communicationClient.createUser();
 
-        firstParticipant = new RoomParticipant().setCommunicationIdentifier(firstParticipantId).setRole(RoleType.ATTENDEE);
-        secondParticipant = new RoomParticipant().setCommunicationIdentifier(secondParticipantId).setRole(RoleType.ATTENDEE);
-        thirdParticipant = new RoomParticipant().setCommunicationIdentifier(thirdParticipantId).setRole(RoleType.CONSUMER);
+        firstParticipant = new RoomParticipant(firstParticipantId);
+        secondParticipant = new RoomParticipant(secondParticipantId);
+        thirdParticipant = new RoomParticipant(thirdParticipantId, ParticipantRole.CONSUMER);
+
+
+        firstParticipant2 = new RoomParticipant(firstParticipantId, null);
```

Comment:
really confusing naming, please either refactor or put track this work in ADO

Reply:
We need to be able to test the two cases: 
`new RoomParticipant(firstParticipantId, null);`
and 
`new RoomParticipant(firstParticipantId);` to make sure both is going to be set as Attendee. I don't think you're actually testing `new RoomParticipant(firstParticipantId, null);`